client-extensions/liferay-object-collection-filter-extension/src/common/components/FilterForm.js

```diff
@@ -1,10 +1,13 @@
 import React from 'react';
+import FilterInput from "./FilterInput";
 
 const FilterForm = ({ selectedFields }) => {
     return (
         <div className="filter-list">
             {selectedFields.map((field) => (
```

Comment:
when you use map method in React, for each item should have a key 
[FYI](https://legacy.reactjs.org/docs/lists-and-keys.html)